java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6804.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "\"@Value\" annotation should inject property or SpEL expression",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6804",
+  "sqKey": "S6804",
+  "scope": "All",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH",
```

Comment:
IMO, `MAINTAINABILITY` could be medium or even low. If you want, we can discuss this further.